src/outline/manager.js

```diff
@@ -415,6 +415,30 @@ export default function (context) {
       .concat(pods)
 
     async function getPods () {
+      async function loadContainerRepresentation (pod) { // namedNode
```

Comment:
- Why load and parse the container rather than fetcher load method?

Reply:
Because you did it in first place and I need access to container Representation.